examples/babel-dynamically-entries/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "serverless-webpack-babel-example",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Babel",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Nicola Peduzzi <thenikso@gmail.com> (http://nikso.net)",
+  "license": "MIT",
+  "devDependencies": {
+    "babel-core": "^6.24.1",
+    "babel-loader": "^7.1.1",
+    "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-polyfill": "6.13.0",
+    "babel-preset-env": "^1.6.0",
+    "serverless-webpack": "file:../../",
+    "webpack": "^1.13.1"
```

Comment:
I've updated all examples dependencies and tested all folders.
It looks like everything is working as expected.